tests/test_custom_scheme_registry.py

```diff
@@ -25,3 +26,11 @@ def test_custom_registry_singleton(entry_points):
     instance2 = custom_schemes_registry()
 
     assert instance1 is instance2
+
+
+def test_custom_registry_url():
+    url1 = to_url("12345", "custom_scheme")
+    url2 = to_url("12345", "custom_scheme2")
+
+    assert url1 == "http://custom/scheme/12345"
+    assert url2 == "http://custom/scheme2/12345"
```

Comment:
I like to put the expected on the left side of "==", so that failing tests shows what the right is missing to be like the left i.e. how it(right side) deviates from correct(left side). The test report is flipped otherwise.